structure_data/structures/7AD3.yaml

```diff
@@ -0,0 +1,13 @@
+auxiliary_protein: ''
+construct: 7ad3
+ligand:
+    chain: I,K
```

Comment:
Shouldn't this only list the chain of the peptide in the binding pocket of the "preferred_chain" receptor?

Reply:
Right, sending a fix